modulo3/astromatch/astromatch/src/Components/AreaCard/AreaCardApp.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from "react";
+import { useState } from "react/cjs/react.development";
+import { getPerfil, choosePerson } from "../../Servicos/Servicos";
+import {
+  ContainerCard,
+  ImageArea,
+  DataMatch,
+  ButtonsArea,
+  ContainerAreaCard,
+  Loader
+} from "./AreaCard.css";
+
+import Coracao from "../../Util/Img/coracaoIcon.svg"
+import CoracaoQuebrado from '../../Util/Img/coracaoIcone2.svg'
+
+export default function AreaCardApp() {
+  const [perfil, setPerfil] = useState(null);
+  const [perfilEscolido, setPerfilEscohido] = useState(true);
+  const [rotacao, setRotacao] = useState(0);
+
+  useEffect(() => {
+    let perfilCompleto = getPerfil();
+
+    perfilCompleto.then((res) => {
+      setPerfil(res.profile);
+    });
```

Comment:
Não é um grande problema agora, mas é uma boa prática sempre manter o catch. Vai que lá no futuro essa API quebra, e quem for consultar pelo surge não entenderá o que está dando errado.